Source/Core/Common/GL/GLInterface/BGL.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/GL/GLContext.h"
+
+class BWindow;
+class BGLView;
+
+class GLContextBGL final : public GLContext
+{
+public:
+  ~GLContextBGL() override;
+
+  bool IsHeadless() const override;
+
+  bool MakeCurrent() override;
+  bool ClearCurrent() override;
+
+  void Update() override;
+
+  void Swap() override;
+
+  void* GetFuncAddress(const std::string& name) override;
+
+protected:
+  bool Initialize(const WindowSystemInfo& wsi, bool stereo, bool core) override;
+
+private:
+  BWindow* m_window;
+  BGLView* m_gl;
+
+  static BGLView* m_current;
```

Comment:
Missed this last time but s_ should be used for static variables (including static member variables):
```suggestion
  static BGLView* s_current;
```
Also, it should be ordered before the non-static members as per the [style guide](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md).